packages/lab/src/preview.ts

```diff
@@ -199,6 +200,29 @@ export class RisePreview extends DocumentWidget<IFrame, INotebookModel> {
     }
   }
 
+  private async _waitForIFrame(iframe: HTMLIFrameElement): Promise<boolean> {
+    const ready = new PromiseDelegate<boolean>();
+
+    const setReady = () => {
+      iframe.contentWindow!.removeEventListener('load', setReady);
+
+      const waitForReveal = setInterval(() => {
+        if (iframe.contentDocument!.querySelector('.reveal')) {
```

Comment:
I would add here the condition that was in the toggle full screen command:

```suggestion
        if (iframe.contentDocument?.readyState === 'complete' && iframe.contentDocument!.querySelector('.reveal')) {
```

Reply:
why? The interval is only set once the iframe is ready. We check before calling setReady.

Reply:
good point I miss that relation ship with the load event.